### PR TITLE
Delete file when exists and overwrite is specified

### PR DESCRIPTION
**Pull request by DoCode**
(see #102)

### CODE REVIEW

**Comment by markwilkie** [comment]
@chcosta - could you take a peek please?

**Comment by DoCode** [comment]
@chcosta sorry it the delay... And yes!
I am on holidays... I look at the comments tonight and come back.

**Comment by DoCode** [comment]
@chcosta please take a look.

**Comment by chcosta** [comment]
Pending clean CI of course.

**Comment by chcosta** [comment]
CI failures appear to be related to your changes, let's take a look at those before getting this merged.

File: src/Microsoft.DotNet.Build.Tasks.IO/src/ZipArchive.cs [hunk 1]
```diff
@@ -60,10 +60,17 @@ public class ZipArchive : Task
...
-                Log.LogError($"Zip file {OutputPath} already exists. Set Overwrite=true to replace it.");
-                return false;
+                if (Overwrite)
+                {
+                    Log.LogMessage(MessageImportance.Low, $"'{OutputPath}' already exists and Overwrite is '{Overwrite}', deleting before zipping...");
+                    IOFile.Delete(OutputPath);
+                }
+                else
+                {
+                    Log.LogError($"Zip file {OutputPath} already exists. Set {nameof(Overwrite)}=true to replace it.");
```
Comment (chcosta) [inline src/Microsoft.DotNet.Build.Tasks.IO/src/ZipArchive.cs]: `return false` got lost

Comment (DoCode) [reply src/Microsoft.DotNet.Build.Tasks.IO/src/ZipArchive.cs]: Yes. I fixed that at the same time...

**Comment by DoCode** [comment]
Ok, I am offline the next 6-8 hours... It's late in Germany. Leave a comment, when other errors occur and I take a look in a few hours. Thanks!

**Pull request merged**